mantidproject/mantid
# TOFTOF GUI: add progress report status message

**Description of work.**
Added a progress bar to the TOFTOF reduction GUI, that appears when 
TOFTOF GUI now shows a progress bar while running the data reduction and After the end of the data reduction a status message appears, indicating the success or failure of the reduction

**To test:**

Start the TOFTOF data reduction GUI: `Interfaces`&rarr;`Direct`&rarr;`DGS Reduction`. Choose facility MLZ and instrument TOFTOF.

Get the test data
[testdata.zip](https://github.com/mantidproject/mantid/files/1356265/testdata.zip)

and make sure, that:
 - the progress bar appears at the bottom of the form when pressing `Reduce`
 - and gets updated during the reduction process
 - and displays a success message, once the reduction has finished
 - or displays a failure message, if the reduction was attempted, but failed (eg. because of invalid data-/vanadium-runs...)


Fixes #22600. 

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
